sale_packaging_default/models/sale_order_line.py

```diff
@@ -27,7 +27,7 @@ def _compute_product_packaging_id(self):
                 line.product_packaging_id = (
                     line.product_id.packaging_ids.filtered_domain(
                         [("sales", "=", True)]
-                    )[:1]
+                    ).sorted("sequence")[:1]
```

Comment:
You know what's weird? The test is correctly written, and the bug is supposed to not exist, but if I remove this change, the test still goes green.

That's what I'd expect because [the default order](https://github.com/odoo/odoo/blob/4adda36260f3ba0fdf75688b419593006b80adb7/addons/product/models/product_packaging.py#L15) is supposed to do automatically what you expect here.

Do you have any other addons installed that alter that default order, or something similar?